api/users/users.proto

```diff
@@ -21,4 +21,5 @@ message User {
 
 message UserReferrer {
 	bytes ethereum_address = 1;
+	bool referrer_valid = 2;
```

Comment:
I have changed my mind and I like the name now. Can you leave a documentation comment above this which lays out the conditions under which this is true or false?

Then regenerate the code. It should be included in the Go file.